build-system/compile/internal-version.js

```diff
@@ -124,8 +124,10 @@ function getVersion() {
     `HEAD~${numberOfCherryPicks}`
   ).slice(0, -2);
 
-  numberOfCherryPicks = String(numberOfCherryPicks).padStart(3, '0');
-  return `${lastCommitFormattedTime}${numberOfCherryPicks}`;
+  return `${lastCommitFormattedTime}${String(numberOfCherryPicks).padStart(
+    3,
+    '0'
+  )}`;
```

Comment:
Nit: instead of inlining the string, can you create a new constant (e.g. `numberOfCherryPicksStr`)?  Otherwise the prettier formatting makes the code hard to read.